.github/workflows/dependency-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Suppressions
+
+on:
+  schedule:
+    # every 3rd hour from 10 through 12
+    - cron: "H 10-12/3 * * *"
```

Comment:
@marvincudjoe - we need to change the cron to run till 12 midnight
H 0,10-23/3 * * *

Reply:
Let us allow the cron job to run at 12 midnight as well - H 0,10-23/3 * * *

Reply:
Done ✅